BAZProject/Utilities/Memento/MovieSave.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  MovieSave.swift
+//  BAZProject
+//
+//  Created by Leobardo Gama Muñoz on 16/02/23.
+//
+
+import Foundation
+
+// MARK: - CareTaker
+public class SaveMovies {
+    let moviesWatched = "MoviesWatched"
+    private let decoder = JSONDecoder()
+    private let encoder = JSONEncoder()
+    private let userDefaults = UserDefaults.standard
+    
+    func save(_ movie: [Result], title: String) throws {
+        let data = try encoder.encode(movie)
+        userDefaults.set(data, forKey: title)
```

Comment:
For the project's scope this is ok. Just making the note that the `UserDefaults` would not be the correct place to save this.